spec/helpers/planning_application_helper_spec.rb

```diff
@@ -31,7 +31,7 @@
     end
 
     context "granted" do
-      let(:planning_application) { create :planning_application, :determined, decision: "granted" }
+      let(:planning_application) { create :planning_application, :determined, decision: "granted", public_comment: "All GDPO compliant" }
```

Comment:
Explicitly setting the `public_comment` in a `let` should only *really* be done if it's being explicitly tested in the included specs. If it's just for creating a valid object, then it should be done in the factory (`spec/factories/planning_application.rb`). Could you add that there, and then remove all these explicit sets?